packages/mobile/src/screens/update-required-screen/UpdateRequiredScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import { Platform, View } from 'react-native'
+
+import IconDownload from 'app/assets/images/iconDownload.svg'
+import { Button, GradientText, Text, useLink } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+export const ANDROID_PLAY_STORE_LINK =
+  'https://play.google.com/store/apps/details?id=co.audius.app'
+export const IOS_APP_STORE_LINK = 'itms-apps://us/app/audius-music/id1491270519'
+
+const messages = {
+  header: 'Update Required',
+  text: 'Behind minimum required version. Do the damn thing.',
+  buttonText: 'Update App'
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  contentContainer: {
+    paddingTop: spacing(32),
+    paddingBottom: spacing(8),
+    paddingHorizontal: spacing(6)
+  },
+  header: {
+    fontSize: 28,
+    lineHeight: 52,
+    textAlign: 'center',
+    textShadowOffset: { height: 2, width: 0 },
+    textShadowRadius: 4,
+    textShadowColor: 'rgba(162,47,235,0.2)'
```

Comment:
no worries at all, but is this a standard value we use for things? might be cool to add text-shadow to our theme? also is this compatible with all theme modes?